docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md

```diff
@@ -0,0 +1,95 @@
+
+## MTL & iGPU & Arc
+
+### Install GPU driver
+
+* Download and Install Visual Studio 2022 Community Edition from the [official Microsoft Visual Studio website](https://visualstudio.microsoft.com/downloads/). Ensure you select the **Desktop development with C++ workload** during the installation process.
+   
+    > Note: The installation could take around 15 minutes, and requires at least 7GB of free disk space.  
+    > If you accidentally skip adding the **Desktop development with C++ workload** during the initial setup, you can add it afterward by navigating to **Tools > Get Tools and Features...**. Follow the instructions on [this Microsoft guide](https://learn.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-170#step-4---choose-workloads)  to update your installation.
+    > 
+    > <img src="https://llm-assets.readthedocs.io/en/latest/_images/quickstart_windows_gpu_1.png" alt="image-20240221102252560" width=100%; />
+
+* Download and install the latest GPU driver from the [official Intel download page](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html). A system reboot is necessary to apply the changes after the installation is complete.
+   
+    > Note: the process could take around 10 minutes. After reboot, check for the **Intel Arc Control** application to verify the driver has been installed correctly. If the installation was successful, you should see the **Arc Control** interface similar to the image provided below
+
+    > <img src="https://llm-assets.readthedocs.io/en/latest/_images/quickstart_windows_gpu_3.png" width=80%; />
+
+* To monitor your GPU's performance and status, you can use either the **Arc Control** application (see the left side of the provided figure) or the **Windows Task Manager** (see the right side of the provided figure):
+    >  <img src="https://llm-assets.readthedocs.io/en/latest/_images/quickstart_windows_gpu_4.png"  width=70%; />
+
+### Setup Python Environment
+
+* Visit [Miniconda installation page](https://docs.anaconda.com/free/miniconda/), download the **Miniconda installer for Windows**, and follow the instructions to complete the installation.
+
+  > <img src="https://llm-assets.readthedocs.io/en/latest/_images/quickstart_windows_gpu_5.png"  width=50%; />
+
+* After installation, open the **Anaconda Prompt**, create a new python environment `llm`:
+  ```bash
+  conda create -n llm python=3.9 libuv
+  ```
+* Activate the newly created environment `llm`:
+  ```bash
+  conda activate llm
+  ```
+ 
+### Install oneAPI 
+
+* With the `llm` environment active, use `pip` to install the **OneAPI Base Toolkit**:
+  ```bash
+  pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0
+  ```
+  
+### Install `bigdl-llm`
+
+* With the `llm` environment active, use `pip` to install `bigdl-llm` for GPU: 
+  ```bash
+  pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
```

Comment:
Shall we use `--extra-index-url` (see https://github.com/analytics-zoo/nano/issues/919)? @Oscilloscope98 

Need to verify it on both Intel and non-Intel network

Reply:
Yes, we can use `--extra-index-url` as the `-f https://developer.intel.com/ipex-whl-stable-xpu` may be deprecated later. We have had some verification before for this `--extra-index-url` on different network. But we could verify them again :)

Reply:
@ivy-lv11 please kindly verify this and update the command.